doc/source/scans/tutorial.rst

```diff
@@ -302,6 +319,25 @@ Perform Fits
   .. figure:: peak.png
      :alt: Fitting a peak
 
+  An alternative 'fitting' method is a "centre of mass" fit. For a set of
+  data points (x, y) it calculates the centre of mass as sum(x*y)/sum(y).
+  The background is subtracted before this calculation is done.
+
+  >>> fit = scan(theta, start=0, stop=2, count=11, fit=CentreOfMass, frames=5, save="centre_of_mass.png")
+  Taking a count at theta=0.00 and two theta=0.00
+  Taking a count at theta=0.20 and two theta=0.00
+  Taking a count at theta=0.40 and two theta=0.00
+  Taking a count at theta=0.60 and two theta=0.00
+  Taking a count at theta=0.80 and two theta=0.00
+  Taking a count at theta=1.00 and two theta=0.00
+  Taking a count at theta=1.20 and two theta=0.00
+  Taking a count at theta=1.40 and two theta=0.00
+  Taking a count at theta=1.60 and two theta=0.00
+  Taking a count at theta=1.80 and two theta=0.00
+  Taking a count at theta=2.00 and two theta=0.00
+  >>> 0.98 <= fit["Centre_of_mass"] <= 1.02
```

Comment:
When I run this test, the Centre of mass comes out closer to about 1.06.  Are you getting something different or did you just run a guess?